pages/common/lpass.md

```diff
@@ -0,0 +1,23 @@
+# lpass
+
+> Command line interface for LastPass Password Manager.
+
+- Login to your LastPass account, by entering your Master Password when prompted:
+
+`lpass login {{username}}`
+
+- Show login status:
+
+`lpass status`
+
+- List all sites grouped by category:
+
+`lpass ls`
+
+- Generate a new password and add to LastPass:
+
+`lpass generate --username {{username}} --url {{gmail.com}} {{gmail.com}} {{password_length}}`
```

Comment:
Probably making the identifier different from the site name would make it clearer. Also we should update the description saying that `Generate a new password for <gmail.com> with identifier <myinbox> and add to LastPass`. That makes it clear what is what.